src/geo/projection/globe_util.js

```diff
@@ -724,37 +730,117 @@ export class GlobeSharedBuffers {
         }
     }
 
-    _createGrid(context: Context) {
-        const gridVertices = new PosArray();
-        const gridIndices = new TriangleIndexArray();
-
-        const quadExt = GLOBE_VERTEX_GRID_SIZE;
-        const vertexExt = quadExt + 1;
-
-        for (let j = 0; j < vertexExt; j++)
-            for (let i = 0; i < vertexExt; i++)
-                gridVertices.emplaceBack(i, j);
-
-        this._gridSegments = [];
-        for (let k = 0, primitiveOffset = 0; k < GLOBE_LATITUDINAL_GRID_LOD_TABLE.length; k++) {
-            const latitudinalLod = GLOBE_LATITUDINAL_GRID_LOD_TABLE[k];
-            for (let j = 0; j < latitudinalLod; j++) {
-                for (let i = 0; i < quadExt; i++) {
-                    const index = j * vertexExt + i;
-                    gridIndices.emplaceBack(index + 1, index, index + vertexExt);
-                    gridIndices.emplaceBack(index + vertexExt, index + vertexExt + 1, index + 1);
+    // Generate terrain grid vertices and indices for all LOD's
+    //
+    // Grid vertices memory layout:
+    //
+    //          First line strip
+    //          ┌───────────────┐
+    //          │┌─────────────┐│
+    // Left     ││┼┼┼┼┼┼┼┼┼┼┼┼┼││ Right
+    // Border   ││┼┼┼┼┼┼┼┼┼┼┼┼┼││ Border
+    // Strip    │├─────────────┤│ Strip
+    //          ││  Main Grid  ││
+    //          │├─────────────┤│
+    //          ││┼┼┼┼┼┼┼┼┼┼┼┼┼││
+    //          ││┼┼┼┼┼┼┼┼┼┼┼┼┼││
+    //          │└─────────────┘│
+    //          ├───────────────┤
+    //          ├───────────────┤
+    //          └───────────────┘
+    //      Bottom strips = Number of LOD's
+    //
+    // export type GridWithLods = {
+    //     gridVertices: PosGlobeExtArray,
+    //     gridIndices: TriangleIndexArray,
+    //     gridSegments: Array<SegmentVector>
+    // };
+    _fillGridMeshWithLods(longitudinalCellsCount: number, latitudinalLods: number[], hasStrip: boolean): GridWithLods {
```

Comment:
Nit: should we eliminate the `hasStrip` flag since it's always true? (I guess this a debugging leftover?)

Reply:
I have removed the flag from the interface, but kept it internal for potential debug uses